app/editor/src/features/admin/report-templates/ReportTemplateFormCharts.tsx

```diff
@@ -41,7 +40,7 @@ export const ReportTemplateFormCharts: React.FC = () => {
         <h2>{values.name}</h2>
         <p>Select the charts that are supported by this template.</p>
       </Col>
-      <Row>
+      {/* <Row>
```

Comment:
Temporarily removed because we currently don't have any configuration options for report templates.